### PR TITLE
Reduce image size

### PR DESCRIPTION
**Pull request by adidenko**
Use 'busybox' instead of 'debian' to reduce docker image size.

### CODE REVIEW

**Comment by dshulyak** [comment]
I am currently using pkill and iproute2 for some testing purposes. Would be nice to have them in image. Is there any way to add them on top of busybox/alpine?

**Comment by adidenko** [comment]
OK, looking into that.

**Comment by adidenko** [comment]
alpine:3.4 has iproute2 package installed, busybox has it too, but let's use alpine with tag pinning.

File: test/e2e/basic_test.go [hunk 1]
```diff
@@ -275,7 +275,7 @@ var _ = Describe("Third party objects", func() {
 		Expect(totalCount).To(BeNumerically("==", len(externalIPs)))
 
 		By("making one of the controllers unreachable and verifying that all ips are rescheduled onto the other pod")
-		rst, _, err := testutils.ExecInPod(clientset, dsPods[0], "pkill", "--echo", "-19", processName)
+		rst, _, err := testutils.ExecInPod(clientset, dsPods[0], "killall", "-19", processName)
 		Expect(err).NotTo(HaveOccurred())
 		Expect(rst).NotTo(BeEmpty())
```
Comment (dshulyak) [inline test/e2e/basic_test.go]: you need to remove it now, and rst should be _ if it is not used

Comment (dshulyak) [inline test/e2e/basic_test.go]: _, _, err = testutils.ExecInPod(clientset, dsPods[0], "killall", "-19", processName)

File: test/e2e/basic_test.go [hunk 2]
```diff
@@ -289,7 +289,7 @@ var _ = Describe("Third party objects", func() {
 		}, 30*time.Second, 1*time.Second).Should(BeNil())
 
 		By("bring back controller and verify that ips are purged")
-		rst, _, err = testutils.ExecInPod(clientset, dsPods[0], "pkill", "--echo", "-18", processName)
+		rst, _, err = testutils.ExecInPod(clientset, dsPods[0], "killall", "-18", processName)
```
Comment (dshulyak) [inline test/e2e/basic_test.go]: _, _, err = testutils.ExecInPod(clientset, dsPods[0], "killall", "-18", processName)

File: test/e2e/basic_test.go (hunk 1 above)
Comment (dshulyak) [inline test/e2e/basic_test.go]: or better make it verbose and leave as is

**Pull request merged**